app/react/src/server/framework-preset-react-docgen.ts

```diff
@@ -28,13 +29,23 @@ export function babel(config: TransformOptions, { typescriptOptions }: Storybook
   };
 }
 
+const tsconfigPath = './tsconfig.json';
+
 export function webpackFinal(config: Configuration, { typescriptOptions }: StorybookOptions) {
-  const { reactDocgen, reactDocgenTypescriptOptions } = typescriptOptions;
+  const { reactDocgen, reactDocgenTypescriptOptions = {} } = typescriptOptions;
 
   if (reactDocgen !== 'react-docgen-typescript') {
     return config;
   }
 
+  if (
+    !reactDocgenTypescriptOptions.tsconfigPath &&
+    !reactDocgenTypescriptOptions.compilerOptions &&
+    fs.existsSync(tsconfigPath)
+  ) {
+    reactDocgenTypescriptOptions.tsconfigPath = tsconfigPath;
```

Comment:
Could this be something that's done internally, inside the plugin? As a user, I prefer tools to default to the standard location/name for a `tsconfig.json` if I don't provide one.

Reply:
Sure!